server/static/registerDescription.json

```diff
@@ -0,0 +1,12 @@
+[   
```

Comment:
@daniellandau How about we use Markdown for the entire content? That would enable us to have links etc. in the content.